app/src/main/java/com/github/geohunt/app/model/PointCalculator.kt

```diff
@@ -0,0 +1,73 @@
+package com.github.geohunt.app.model
+
+import com.github.geohunt.app.model.database.api.Location
+import kotlin.math.*
+
+/**
+ * Object giving access to its only method computePoints, which allows the user to compute
+ * the amount of points a claim gets.
+ */
+object PointCalculator {
+    /**
+     * Hyper parameters of the function used to compute the points
+     */
+    private const val MEAN = 0.0
+    private const val STD = 0.15
+
+    /**
+     * The least amount of points possible.
+     * Note that the function used can't be negative so negative MIN_POINTS value
+     * will work the same way as MIN_POINTS = 0
+     */
+    const val MIN_POINTS = 0.0
+
+    /**
+     * The maximum amount of points possible
+     */
+    const val MAX_POINTS = 5000.0
+
+    /**
+     * Utility definitions
+     */
+    private val calculator = gaussianDistributionPDF(MEAN, STD)
+    private val fctMaxValue = calculator(MEAN)
+    private val ratio = MAX_POINTS / fctMaxValue
+
+    /**
+     * Main function of the PointCalculator, computes the amount of points a claim is worth.
+     * Uses a gaussian distribution and the distance between the two Locations to compute
+     * the points.
+     * @param l1 First location used to compute points
+     * @param l2 Second location used to compute points
+     */
+    fun computePoints(l1: Location, l2: Location): Double {
+        val points = calculator(l1.distanceTo(l2)) * ratio
+        return clamp(MIN_POINTS, points, MAX_POINTS)
+    }
```

Comment:
I believe it would be better if we did not hard code this here and instead create a `interface`. However we are still early in the project, as such I guess we could create a user story for having more type of "score system" for instance : 
* All or nothing (must be in a 10m radius)
* Gaussian like 

In this user story we could also include other parameters such as 
* Time it took until resolution
* ...